### DISCUSSION TITLE
Osu! like list

### DISCUSSION THREAD

**Discussion by Ruttie2006** [AvaloniaUI/Avalonia#9977] [Q&A] [answered]
I want to create a list like the Osu! beatmap list.
Does anyone have any ideas on how to do this?
(For context, I want to have selectable items, where the size of the item increases when you click on it)
I currently have the following:
![image](https://user-images.githubusercontent.com/80394237/212483192-2abd9526-ffa4-4350-8894-4f74da3cb359.png)
but it doesn't adapt to a change in the windows size.

**Comment by maxkatz6** [comment, marked as answer]
I was interested to try this kind of menu. Pushed a repository here https://github.com/maxkatz6/AvaloniaOsuMenuExample/tree/main
Didn't have any problems with window size though. Can you add more details what went wrong in your case?
